user/tests/wiring/api/spi.cpp

```diff
@@ -209,11 +209,15 @@ test(spi_hal_backwards_compatibility)
     SPI_Mode mode = SPI_MODE_MASTER;
     HAL_SPI_TransferStatus status;
     HAL_SPI_AcquireConfig config;
+    hal_spi_config_t spi_config = {};
+    spi_config.size = sizeof(spi_config);
+    spi_config.version = HAL_SPI_CONFIG_VERSION_1;
```

Comment:
Prefer to use latest (`HAL_SPI_CONFIG_VERSION`). It's fine if some fields are left as defaults (zero-filled), that should be correctly handled.